library/src/animationManager.cxx

```diff
@@ -23,7 +23,7 @@ void animationManager::Initialize(
   this->Playing = false;
   this->CurrentTime = 0;
   this->CurrentTimeSet = false;
-
+  this->AnimationIndex = -1;
```

Comment:
I think the default in the options layer is 0, better to stay coherent imo.

Reply:
https://github.com/f3d-app/f3d/blob/281b17ab2b2384fbc652cb7c79f6b8808342e9a4/library/src/animationManager.cxx#L94C3-L99C6

It will affect some logic within initialize function if it is modified to 0?

Reply:
thode code you link is not about initializing, it is about enabling all the animations if animationIndex == -1. not sure to follow.

Reply:
I guess I am not understanding where the -1 originates for AnimationIndex. I thought it only comes from when I assign it that value through initialization. After looking again, I'm seeing that `options->getAsInt("scene.animation.index")` can also contain that value.